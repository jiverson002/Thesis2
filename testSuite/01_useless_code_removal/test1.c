int main() {
   int a;
   a = 11;
   if (1 < 4) {
      a = 12;
   } else {
      a = 13;
   }
   return a;
}
