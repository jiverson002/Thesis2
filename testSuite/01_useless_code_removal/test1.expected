int main() {
   int a;
   if (1 < 4) {
      a = 12;
   } else {
      a = 13;
   }
   return a;
}
